### ISSUE TITLE
Remove duplication between AWS and GCP subprojects

### ISSUE THREAD

**Issue by marcospereira** [lightbend/akka-cloud-platform-deploy#18]
There is a number of places where we can remove duplications. For example, the `config.ts` and `telemetry.ts` files.

One option is to go back to the original idea of having a single code base and use `pulumi config set` to choose a cloud provider.

**Comment by RayRoestenburg** [comment]
I would prefer to have duplication. This codebase is for examples and users only very likely use one of the platforms. One codebase with less shared is easier to modify.

**Comment by marcospereira** [comment]
> I would prefer to have duplication. This codebase is for examples and users only very likely use one of the platforms. One codebase with less shared is easier to modify.

Yeah, something to find a balance, I suppose. While it is good to have this oriented to customers to modify, it is bad to have duplicated files such as `telemetry.ts`.
